hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
+
+    
+    
+    
+if name == "__name___"
```

Comment:
Syntax error, should be `if __name__ == "__main__":`

`if __name__ == "__main__":`